website/docs/reference/appsmith-framework/widget-actions/show-modal.md

```diff
@@ -1,31 +1,22 @@
 ---
 sidebar_position: 3
+description: Open a Modal widget using the showModal() Appsmith framework function.
 ---
-# Show Modal
 
-This function is used to open a Modal when triggered.
+# `showModal()`
 
-
-<figure>
-  <img src="/img/show-modal-action.png" style= {{width:"700px", height:"auto"}} alt="open a Modal"/>
-  <figcaption align = "center"><i>Open a Modal</i></figcaption>
-</figure>
-
-
-
-
-:::tip
-Remember to create a new Modal before calling this action
-:::
+This page provides information for using the `showModal()` framework function to open a Modal widget.
```

Comment:
Actually we should change this line and used it to just describe the function and what it is used for.